src/main/java/net/rptools/maptool/model/Token.java

```diff
@@ -1738,6 +1738,18 @@ public Object getState(String property) {
     return state.get(property);
   }
 
+  public List<String> getSetStates() {
```

Comment:
For me the method name is a bit confusing because getSet. Maybe getActiveStates or something similar would be better?

Reply:
I wasn't totally happy with it either, but wasn't coming up with anything better at the time.